### PR TITLE
feature: add support for multi-architecture image

### PR DESCRIPTION
**Pull request by kyujin-cho**

### CODE REVIEW

File: src/lib/backend.ai-client-node.ts [hunk 1]
```diff
@@ -1060,7 +1062,7 @@ class Client {
 
   // legacy aliases (DO NOT USE for new codes)
   createKernel(kernelType, sessionId = undefined, resources = {}, timeout = 0) {
-    return this.createIfNotExists(kernelType, sessionId, resources, timeout);
+    return this.createIfNotExists(kernelType, sessionId, 'x86_64', resources, timeout);
```
Comment (lizable) [inline src/lib/backend.ai-client-node.ts]: I'm not sure whether it's a good idea to use hard-coded `x86_64` value here.
How about adding key-value in config file instead?

File: src/lib/backend.ai-client-node.js [hunk 2]
```diff
@@ -969,7 +971,7 @@ class Client {
     }
     // legacy aliases (DO NOT USE for new codes)
     createKernel(kernelType, sessionId = undefined, resources = {}, timeout = 0) {
-        return this.createIfNotExists(kernelType, sessionId, resources, timeout);
+        return this.createIfNotExists(kernelType, sessionId, 'x86_64', resources, timeout);
     }
```
Comment (lizable) [inline src/lib/backend.ai-client-node.js:975]: Same here.

**Review by lizable — CHANGES_REQUESTED** [review]
It works okay, but I have one thing to suggest.
Please check the comment below.

File: src/lib/backend.ai-client-node.ts (hunk 1 above)
Comment (inureyes) [reply src/lib/backend.ai-client-node.ts]: Also, the parameter order should be kept for compatibility. @kyujin-cho

Comment (inureyes) [reply src/lib/backend.ai-client-node.ts]: Fixed.

Comment (inureyes) [reply src/lib/backend.ai-client-node.ts]: @lizable It is for legacy code. So only x86_64 will come into the role.

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**